Mathlib/Probability/Moments/ComplexMGF.lean

```diff
@@ -297,4 +298,42 @@ lemma eqOn_complexMGF_of_mgf [IsProbabilityMeasure μ]
 
 end EqOfMGF
 
+section ext
+
+variable {Ω' : Type*} {mΩ' : MeasurableSpace Ω'} {Y : Ω' → ℝ} {μ' : Measure Ω'}
+
+/-- Exponential map onto the circle, defined as additive character -/
+noncomputable
+def probFourierChar : AddChar ℝ Circle where
+  toFun z := Circle.exp (z)
+  map_zero_eq_one' := by simp only; rw [Circle.exp_zero]
+  map_add_eq_mul' x y := by simp only; rw [Circle.exp_add]
```

Comment:
You could rename that `Circle.expAddChar` and move it to the Analysis/Complex/Circle file, since it's very similar to `Circle.expHom`. Put it just before the `variable` line that you added?

The next lemma `probFourierChar_ne_one` (to be renamed `expAddChar_ne_one`) can't be moved there though, since that file does not import enough to know about `Real.pi` (and we should probably not add too many imports there).